go.mod

```diff
@@ -0,0 +1,13 @@
+module gomodules.xyz/union-logr
+
+go 1.12
+
+require (
+	github.com/go-logr/glogr v0.1.0
```

Comment:
I don't like that this has dependency on nats because of the example package. We should only have dependency on logr package.